File: Module 3 - Functions/parametres and Arguments/ReturnTypes/Returntypes.py
"""
Python don't needed returntypes it automatically decide type of Function.
To print return as an output 
* Print(functioncall)
"""

def get_age():
    return 20

def get_price():
    return 99.50

def get_grade():
    return 'A'

def get_name():
    return "X-Creator"

def is_passed():
    return True

def show_message():
    print("Welcome to Python Functions")


age = get_age()
price = get_price()
grade = get_grade()
name = get_name()
passed = is_passed()

show_message()

print("Name:", name)
print("Age:", age)
print("Price:", price)
print("Grade:", grade)
print("Passed:", passed)

File: Module 3 - Functions/parametres and Arguments/ReturnTypes/Returntypes.cpp
/* Returntypes decide which datatypes use function for argument and return.
To print return as an output 
* cout <<functioncall;
*/

#include <iostream>
#include <string>
using namespace std;

// int → returns an integer
int getAge()
{
    return 20;
}

// double → returns a decimal value
double getPrice()
{
    return 99.50;
}

// char → returns a single character
char getGrade()
{
    return 'A';
}

// string → returns text
string getName()
{
    return "X-Creator";
}

// bool → returns true or false
bool isPassed()
{
    return true;
}

// void → returns nothing
void showMessage()
{
    cout << "Welcome to C++ Functions\n";
}

int main()
{
    int age = getAge();
    double price = getPrice();
    char grade = getGrade();
    string name = getName();
    bool passed = isPassed();

    showMessage();

    cout << "Name: " << name << endl;
    cout << "Age: " << age << endl;
    cout << "Price: " << price << endl;
    cout << "Grade: " << grade << endl;
    cout << "Passed: " << passed << endl;

    return 0;
}